testing/python/language/test_tilelang_language_warp_vote.py

```
@@ -7,6 +7,7 @@
 T.ballot_sync       – __ballot_sync→uint64 (CUDA, zero-ext) / __ballot uint64 (HIP, all lanes)
 T.ballot            – full-warp ballot_sync / __ballot uint64 (HIP, all lanes)
 T.activemask        – __activemask→uint64 (CUDA, zero-ext) / __ballot(1) uint64 (HIP, all lanes)
+T.__ffs             – __ffs / __ffsll (CUDA)
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)